packages/services/f-feature-management/test/data/schema.json

```diff
@@ -0,0 +1,221 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "type": "object",
+    "required": [ "features", "createdAt" ],
+    "additionalProperties": false,
+    "properties": {
+        "createdAt": {
+            "type": "string",
+            "format": "date-time"
+        },
+        "features": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": ["key", "valueType", "defaultValueRules"],
+                "additionalProperties": false,
+                "properties": {
+                    "key": {
+                        "type": "string",
+                        "description": "The unique key of the feature."
+                    },
+                    "valueType": {
+                        "type": "string",
+                        "enum": ["string", "bool", "int"],
+                        "description": "The type of the value held by the feature."
+                    },
+                    "defaultValueRules": {
+                        "type": "array",
+                        "description": "Default values in case nothing matches in evaluation.",
+                        "items": {
+                            "type": "object",
+                            "required": ["restrictions", "value"],
+                            "additionalProperties": false,
+                            "properties": {
+                                "restrictions": {
+                                    "$ref": "#/definitions/predicate"
+                                },
+                                "value": {
+                                    "$ref": "#/definitions/featureValue",
+                                    "description": "Default value."
+                                }
+                            }
+                        }
+                    },
+                    "evaluationLogic": {
+                        "type": "object",
+                        "description": "Information required to evaluate the value.",
+                        "required": ["type", "detail"],
+                        "additionalProperties": false,
+                        "properties": {
+                            "type": {
+                                "type": "string",
+                                "enum": ["IdHash"],
+                                "description": "The mechanism used for evaluation."
+                            },
+                            "detail": {
+                                "type": "object",
+                                "description": "Can vary depending on 'type'. Only IdHash defined at the moment.",
```

Comment:
I'm assuming that when/if more than `IdHash` is defined then there'll be some schema validation based around a `oneOf` structure to mandate certain property requirements depending on which type is chosen?

Reply:
Yes, that's correct. The existing schema only enforces the requirements for `IdHash`, and we will rework the schema if/when a new type is introduce (which will be infrequently, if ever).